criu/pstree.c

```diff
@@ -397,7 +397,7 @@ static int prepare_pstree_for_shell_job(pid_t pid)
 		return 0;
 
 	old_gid = root_item->pgid;
-	if (old_gid != current_gid) {
+	if (old_gid != current_gid && old_gid != current_sid) {
```

Comment:
> So 10,10 switched to 20,21, it does not feel right to me, better if restore would just ignore 21 in such case.

This doesn't convince me. If I remember right, each shell job has own process group. It means that with this change, criu restores process in another shell job?

Reply:
Ok, looks like I was wrong -> will close. Thank you!